INSTRUCTION.md

```diff
@@ -0,0 +1,17 @@
+how to run MySQL container with a volume attached
+docker run -d -p 3306:3306 --name mysql-local \
+-e MYSQL_ROOT_PASSWORD=1234 \
+-e MYSQL_USER=app_user \
+-e MYSQL_PASSWORD=1234 \
+-e MYSQL_DATABASE=app_db \
+-v mysql-volume:/var/lib/mysql \
+mysql-local:1.0.0
```

Comment:
Ensure that the image name `mysql-local:1.0.0` matches the correct image tag you have built or pulled. This is crucial for the container to run successfully.